src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -182,20 +207,30 @@ public CourseDetailsBundle getCourseDetails(String courseId) throws EntityDoesNo
         return courseDetailsList;
     }
 
+    /**
+     * @param courseId
+     * @return a list of section names for a course using the courseId
+     * @throws EntityDoesNotExistException
+     */
     public List<String> getSectionsNameForCourse(String courseId) throws EntityDoesNotExistException {
         return getSectionsNameForCourse(courseId, false);
     }
 
+    /**
+     * @param course
+     * @return a list of section names for a course using the {@link CourseAttributes}
+     * @throws EntityDoesNotExistException
+     */
     public List<String> getSectionsNameForCourse(CourseAttributes course) throws EntityDoesNotExistException {
         Assumption.assertNotNull("Course is null", course);
         return getSectionsNameForCourse(course.getId(), true);
     }
     
     /**
-     * Get list of section names for a course with or without a need to check if the course is existent
      * @param courseId Course ID of the course
      * @param hasCheckIsPresent Determine whether it is necessary to check if the course exists
```

Comment:
Though this wasn't due to your changes, could you also help fix this by changing this to `isCourseVerified` as well?